LSP-svelte.sublime-settings

```diff
@@ -119,6 +119,10 @@
 			}
 		}
 	},
+	"file_watcher": {
+		"patterns": ["{**/*.js,**/*.ts,**/*.json}"],
+		"events": ["create", "change"]
```

Comment:
Is `*.json` necessary? The original server doesn't seem to do that https://github.com/sveltejs/language-tools/blob/e3951f82599f370e3bbba56116670013affd13d1/packages/svelte-vscode/src/extension.ts#L144-L144

Also, the original watcher also enables `delete` event.

Reply:
Also it has this custom message on ts/js files being modified - https://github.com/sveltejs/language-tools/blob/e3951f82599f370e3bbba56116670013affd13d1/packages/svelte-vscode/src/extension.ts#L311-L328

Not sure why it needs it if it also has the watcher...

Reply:
OK, so custom notification seems like a more efficient version of the watcher but functionality-wise it should work the same.